develop/components/pages/DatepickerPage.tsx

```diff
@@ -1,50 +1,84 @@
+                    </div>
+                </div>
+
+                <div className="info">
+                    <h2>Output</h2>
+                    <p>Here are sample outputs</p>
+                    <div className="result">
+                        <Datepicker
+                            name="datepicker"
+                            value={datepicker}
+                            onChange={setDatepicker}
+                            minDate={new Date("1970-10-10")}
+                            maxDate={new Date("2022-10-10")}
+                            label="Datepicker label"
+                        />
+                    </div>
+
+                    <p>With custom locale</p>
+                    <div className="result">
+                        <Datepicker
+                            name="datepicker"
+                            value={datepicker2}
+                            onChange={setDatepicker2}
+                            minDate={new Date("1970-10-10")}
+                            maxDate={new Date("2022-10-10")}
+                            label="Swedish date"
+                            locale="sv-se"
+                        />
                     </div>
 
-                    <div className="info">
-                        <h2>Output</h2>
-                        <p>Here are sample outputs</p>
-                        <div className="result">
-                            <Datepicker
-                                name="datepicker"
-                                value={this.state.datepicker}
-                                onChange={(date) => this.setState({ datepicker: date })}
-                                minDate={new Date("1970-10-10")}
-                                maxDate={new Date("2022-10-10")}
-                                label="Datepicker label"
-                                placeHolder="dd/mm/yyyy"
-                            />
-                        </div>
+                    <p>With custom custom calendar icon and clearable</p>
+                    <div className="result">
+                        <Datepicker
+                            name="datepicker"
+                            value={datepicker3}
+                            onChange={setDatepicker3}
+                            minDate={new Date("1970-10-10")}
+                            maxDate={new Date("2022-10-10")}
+                            calendarIcon={moreIcon}
+                            clearable={true}
+                            showLeadingZeros={false}
+                        />
                     </div>
 
+                    <p>Disabled</p>
+                    <div className="result">
+                        <Datepicker
+                            name="datepicker"
+                            value={datepicker3}
+                            onChange={setDatepicker3}
+                            minDate={new Date("1970-10-10")}
```

Comment:
you should declare a min and max date at the beginning of this component and just pass it everywhere you need them.

Reply:
Done